Sources/NetworkProtection/Storage/NetworkProtectionTunnelHealthStore.swift

```diff
@@ -62,8 +62,8 @@ public final class NetworkProtectionTunnelHealthStore {
             guard newValue != userDefaults.bool(forKey: Self.isHavingConnectivityIssuesKey) else {
                 return
             }
-#if os(macOS)
             userDefaults.set(newValue, forKey: Self.isHavingConnectivityIssuesKey)
```

Comment:
I didn’t actually mean to commit this… but it seems like a mistake that it wasn’t setting UserDefaults anyway… thoughts?